.github/workflows/pythonpackage.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Run tox
       run: tox -e pep8
 
-  py37:
+  py37-ubuntu:
```

Comment:
Can we turn this into a matrix like urllib3? https://github.com/urllib3/urllib3/blob/82d1f1833e9880701f72ad9a50155e20dcc60efd/.github/workflows/ci.yml#L28-L37 I feel that would reduce duplication and allow us to more easily add supported python versions and operating systems.

Reply:
Will do